src/views/Header/Header.js

```diff
@@ -479,10 +479,13 @@ class Header extends React.PureComponent {
       safeHeaderStyle,
     ];
 
+    const { safeAreaViewProps } = options;
+
     return (
       <SafeAreaView
         forceInset={{ top: 'always', bottom: 'never' }}
         style={containerStyles}
+        {...safeAreaViewProps}
```

Comment:
I thought about this some more and I think we probably don't want people to inject any random prop in here, they could accidentally break the layout in their app. I think instead we should only pay attention to the `forceInset` prop. so we would change it to something like the following:

```
const { safeAreaViewProps = {} } = options;
const forceInset = safeAreaViewProps.forceInset || { top: 'always', bottom: 'never' }; 

return (
  <SafeAreaView
    forceInset={forceInset}
    style={containerStyles}
  >
```

Reply:
Maybe it makes more sense to have the prop named as `headerForceInset` instead of `safeAreaViewProps` since we're not passing all props here and `SafeAreaView` is an implementation detail.

Reply:
or we can put safe header style inside that too.
Does that sound right?
So that all the new props we add in the future we can add under that object.

Reply:
@satya164 - makes sense. let's do what @satya164 suggested